website/docs/docs/introduction.md

```diff
@@ -11,6 +11,36 @@ dbt compiles and runs your analytics code against your data platform, enabling y
 
 Read more about why we want to enable analysts to work more like software engineers in [The dbt Viewpoint](/community/resources/viewpoint).
 
+<Lightbox src="/img/docs/cloud-overview.jpg" width="60%" title="dbt works alongside your ingestion, visualization, and other data tools, so you can transform data directly in your cloud data platform." />
+
+## dbt products
+
+You can use dbt to quickly and collaboratively transform data and deploy analytics code following software engineering best practices like version control, modularity, portability, CI/CD, and documentation. This means anyone on the data team can safely contribute to production-grade data pipelines.
+
+To cater to the diverse needs of data teams, dbt offers two primary solutions:
+
+- [**dbt Cloud**](#dbt-cloud) &mdash; A comprehensive, full managed service enabling teams to develop, test, deploy, and explore data products safely and reliably.
+- [**dbt Core**](#dbt-core) &mdash; An open-source CLI tool that enables data teams to transform data using analytics engineering best practices.
```

Comment:
I wonder if we should use "suitable for users who prefer manual setup" here like you do on the GS page. I love that framing because it's super helpful